styles.css

```diff
@@ -90,6 +90,61 @@ button {
 
 nav {
   display: none;
+  flex-direction: column;
+  justify-content: space-between;
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  right: 0;
+  background-color: #fff;
+}
+
+.nav-on {
+  display: flex;
+}
+
+.navtop {
+  height: 115px;
+  padding-right: 20px;
+  padding-top: 25px;
+  background: top left url(images/image_geometry_menu_1.svg) no-repeat;
+  text-align: right;
+}
+
+.navbottom {
+  height: 150px;
+  background: bottom left url(images/image_geometry_menu_2.svg) no-repeat;
+}
+
+.main-nav {
+  flex: calc(100% - 265px);
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.main-nav li {
+  margin: 0;
+  padding: 30px 25px 0 25px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 300%;
+  text-align: center;
+}
+
+.main-nav li a {
+  padding: 5px 0;
+  display: block;
+  text-decoration: none;
+  color: #1a2236;
+}
```

Comment:
- [ ] Kindly add `font-weight: bold` to the mobile menu links as it's requested to stick to the Figma design as much as possible in the project requirement.
### Your design :point_down: 
![Screenshot from 2023-06-20 08-28-59](https://github.com/balatstar/portfolio/assets/91142432/a90082da-d7dd-4e8a-9aae-d44de6ebd840)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-06-20 08-29-39](https://github.com/balatstar/portfolio/assets/91142432/ed25795a-4da3-4134-a24d-d3246e0d4de4)